src/audio/volume/volume.c

```diff
@@ -315,6 +333,9 @@ static inline void volume_ramp(struct processing_module *mod)
 		cd->volume[i] = new_vol;
 	}
 
+	for (i = cd->ramp_channel_counter; i < cd->channels; i++)
+		cd->volume[i] = cd->volume[0];
```

Comment:
can we add some explanatory note here? since it seems only for the volume of every channels is the same.

Reply:
@andrula-song , new comments added.